tests/testthat/test-_bugs.R

```diff
@@ -86,12 +86,12 @@ test_that("Duplicated labels",
             Q5r_2 = structure(c(2, 2, 4, 5, 5, 5, 4, 5, 5, 3, 5, 3, 5,
             4, 3, 4, 3, 4, 5, 5), questiontype = "NumberMulti", name = "Q5r_2", label = "Landscaping", question = "How satisfied were you with... 2")), row.names = c(NA,
         20L), class = "data.frame")
-
-    expect_warning(print(Regression(Q4_2~.,, data = df.small, show.labels = TRUE,
+    test.formula <- as.formula(paste0("Q4_2 ~ ", paste0(names(df.small)[-1], collapse = " + ")))
```

Comment:
Why was the `.` removed from these tests?

Reply:
I don't recall exactly why I did that for this case. I did go through a lot of test files and tidied up some things. It doesn't look necessary in this case and the tests still pass when the `.` is restored in the formula. I'll restore it in a commit that addresses this and the other comments too.